GraphQL-README.md

```diff
@@ -135,6 +136,67 @@ query BrandByTagQuery($tag: String!) {
 }
 ```
 
+## individual bank by name
+
+**query**
+
+```graphql
+query BrandByTagQuery($name: String!) {
+  brand(name: $name) {
+    tag
+    name
+    website
+    commentary {
+      rating
+      embrace
+      fromTheWebsite
+      ourTake
+      amountFinancedSince2016
+      fossilFreeAlliance
+      top_pick
```

Comment:
Could you update to `topPick` while you're in here and same thing in the example for brand by tag above